x/gov/testutil/expected_keepers.go

```diff
@@ -26,6 +26,13 @@ type BankKeeper interface {
 	bankkeeper.Keeper
 }
 
+// PoolKeeper extends the gov's actual expected PoolKeeper.
+type PoolKeeper interface {
+	types.PoolKeeper
```

Comment:
Hum, that defeats a bit the point of having an expected keeper interface.
Why is it needed for mocks?

Reply:
removed, missed while cleaning up